base/db/R/get.trait.data.R

```diff
@@ -56,37 +63,49 @@ get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon,
   file.remove(old.files)
 
   # find appropriate pft
-  if (is.null(modeltype)) {
-    pftid <- db.query(
-      query = paste0("SELECT id FROM pfts WHERE name='", pft$name, "'"),
-      con = dbcon
-    )[['id']]
-  } else {
-    pftid <- db.query(
-      query = paste0(
-        "SELECT pfts.id FROM pfts, modeltypes WHERE pfts.name='", pft$name,
-        "' and pfts.modeltype_id=modeltypes.id and modeltypes.name='", modeltype,
-        "'"
-      ),
-      con = dbcon
-    )[['id']]
+  pftres <- (dplyr::tbl(dbcon, "pfts")
+    %>% dplyr::filter(name == pft$name))
+  if (!is.null(modeltype)) {
+    pftres <- (pftres %>% dplyr::semi_join(
```

Comment:
Does this need to be enclosd in quotes? Is there any effect of doing so?

Reply:
Do you mean enclosed in parentheses? It makes no difference here, I just got in the habit of wrapping my pipelines so that the `%>%` could be at the beginning of the line instead of the end.

Reply:
I'd vote for sticking w/ convention of pipes at end of line and not using parentheses (though unlike spaces, not enough to insist on, or hold up this PR)